widget/embedded/src/hooks/useFetchQuote.ts

```diff
@@ -1,38 +1,51 @@
-import type { BestRouteRequest, BestRouteResponse } from 'rango-sdk';
+import type { RequestOptions } from 'https';
```

Comment:
You imported the type from the incorrect package.

Reply:
fixed